contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,462 @@
+```
+
+```iOS
+posthog.identify("distinct_id", 
+          properties: ["name": "Max Hedgehog", "email": "max@hedgehogmail.com"])
+```
+
+</MultiLanguage>
+
+## How `identify` works
+
+When a user starts browsing your website or app, they'll be automatically assigned an **anonymous ID**, which is then stored locally. This allows us to track anonymous users – even across different sessions.
+
+> Note that depending on your [persistence configuration](/docs/libraries/js#persistence), the anonymous ID may not be stored, and thus regenerated across sessions.
+
+By calling `identify` with a `distinct_id` of your choice (usually the user's ID in your database, or their email), you are linking the anonymous ID and distinct ID together.
+
+Thus all past and future events made with that anonymous ID are now associated with the distinct ID. This enables you to do things like associate events with a user from before they log in for the first time, or associate their events across different devices or platforms.
+
+> **A note on using `identify` in the backend:**
+>
+> Although it is technically possible to call `identify` using our backend SDKs, it is typically most useful in frontends. This is because there is no concept of anonymous sessions in the backend SDKs.
+
+## Best practices when using `identify`
+
+### 1. Call `identify` as soon as you're able to
+
+In your frontend, you should call identify as soon as you're able to. Typically, this is every time your app loads for the first time, or directly after your users log in. This ensures that events sent during your users' sessions are correctly associated with them. 
+
+You only need to call `identify` once per session.
+
+### 2. Use unique strings for Distinct IDs
+
+If two users have the same distinct ID, their data will be merged and they will be considered as one user in PostHog. Two common ways in which this can happen are:
+
+- Your logic for generating IDs does not generate sufficiently strong IDs and you can end up with a clash where 2 users have the same ID.
+- There's a bug, typo, or mistake in your code leading to most or all users being identified with generic IDs like `null`, `true`, or `distinctId`.
+
+PostHog also has a few built-in protections to stop the most common distinct ID mistakes:
+
+- We do not allow identifying users with empty space strings of any length – e.g.,`' '`, `'       '`, etc.
+- We do not allow identifying users with the following strings (case insensitive):
+    - `anonymous`
+    - `guest`
+    - `distinctid`
+    - `distinct_id`
+    - `id`
+    - `not_authenticated`
+    - `email`
+    - `undefined`
+    - `true`
+    - `false`
+
+- We do not allow identifying users with the following strings (case sensitive):
+  - `[object Object]` 
+  - `NaN`
+  - `None`
+  - `none`
+  - `null`
+  - `0`
+
+- We do not allow identifying a user that has already been identified with a different distinct ID. For example:
+
+<MultiLanguage selector="tabs">
```

Comment:
This is too much detail for the main section - move it to troubleshooting / FAQ

Reply:
hmm I see your point, but this feels like more of a nit. It fits in nicely to this section, I would prefer to leave here